Trippy/View Controllers/Templates/MapItemListViewController.m

```diff
@@ -7,14 +7,29 @@
 
 #import "MapItemListViewController.h"
 #import "CacheDataHandler.h"
+#import "JHUD.h"
 
 @interface MapItemListViewController () <UITableViewDelegate, UITableViewDataSource, CacheDataHandlerDelegate>
+@property (weak, nonatomic) IBOutlet UIView *loadingView;
 @property (strong, nonatomic) UIRefreshControl *refreshControl;
 @property (strong, nonatomic) CacheDataHandler *handler;
 @end
 
 @implementation MapItemListViewController
 
+- (void)viewWillAppear:(BOOL)animated {
+    JHUD *hudView = [[JHUD alloc] initWithFrame:self.loadingView.bounds];
+    hudView.messageLabel.text = @"Loading your trip info...";
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"paper_plane" ofType:@"gif"];
+    NSData *data = [NSData dataWithContentsOfFile:path];
+    hudView.indicatorViewSize = CGSizeMake(200, 200); // Maybe you can try to use (100,250);😂
```

Comment:
What is that comment? Also, a comment with an emoji? ... I mean, I guess it's supported... 🤔